ui/src/authentication/pages/Login/Login.js

```diff
@@ -94,7 +97,7 @@ class LoginPage extends Component {
                     >
                       <Grid.Row>
                         <Grid.Column stretched width={8} textAlign="right">
-                          <Header size="massive" className="inline-block">
+                          <Header size="huge" className="inline-block">
```

Comment:
sorry about that, can you move massive as a name of a class instead? because this change will influence the looks of the login page. I guess that was the spot where there was console warning? if yes then semantic acts a bit inconsistent here, it displays a warning but adds to the class of the element anyway.

Reply:
done!